tests/integration/api_v1/http/transactions.test.js

```diff
@@ -269,12 +267,15 @@ describe('Transactions API', () => {
 
 	describe('Retrieve transaction list within timestamps', () => {
 		it('transactions within set timestamps are returned', async () => {
-			const from = moment(refTransaction.timestamp).subtract(1, 'day').unix() * (10 ** 3);
-			const toTimestamp = refTransaction.timestamp;
+			const transactions = (await api.get(`${endpoint}`)).data;
```

Comment:
Is there any reason to use `${endpoint}` instead of just endpoint?

Reply:
Updated it to `endpoint`. Thanks!